component/src/test/resources/conf/security/permissions.yml

```diff
@@ -0,0 +1,141 @@
+################################################################################
+#   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Wrong licence year

Reply:
Fix for all below occurrences.

Reply:
Fixed in 6bf095af9d7a4c00220069badb50517cec553139